packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -542,6 +546,7 @@ export function generateGrid(
                               filterable: "false",
                               type: '"actions"',
                               align: '"right"',
+                              pinned: `"${actionsColumnPinned}"`,
```

Comment:
Should it be possible to disable pinning the action column?

Reply:
I don't think we'll ever need this...